terraform-plans/templates/github/snap_release.yaml.tftpl

```diff
@@ -10,15 +10,19 @@ on:
     branches: [ ${branch} ]
   release:
     types: [ published ]
+  workflow_dispatch:
```

Comment:
If you include `workflow_dispatch`, then should it make more sense to force `check`? Because one can do a release without `check`, that's not ideal

Reply:
In the case of [charmed-openstack-exporter-snap](https://github.com/canonical/charmed-openstack-exporter-snap) there is no `check` to do because it's just a wrapper to package the code from upstream.

Reply:
In any case, I've removed the if conditions and now it always will check first.

Reply:
Thanks! I didn't notice that there's only a `snapcraft.yaml` in it. In any case, I think corner case will happen in the future, it's better to have some general pattern to fix it

Reply:
Ok, we can look into adding the release.yaml to automation later :+1: